bin/clover/src/pipeline-steps/generateAssetFuncs.ts

```diff
@@ -205,27 +198,7 @@ function generatePropBuilderString(
           generatePropBuilderString(prop.typeProp, indent_level + 1)
         }\n` +
         `${indent(indent_level)})\n`;
-
-      return `new PropBuilder()\n` +
-        `${indent(indent_level)}.setKind("${prop.kind}")\n` +
-        `${indent(indent_level)}.setName("${prop.name}")\n` +
-        `${indent(indent_level)}.setHidden(${prop.data?.hidden ?? false})\n` +
-        `${
-          generateWidgetString(
-            prop.data?.widgetKind,
-            is_create_only,
-            indent_level,
-          )
-        }` +
-        `${
-          prop.data?.defaultValue
-            ? `${indent(indent_level)}.setDefaultValue(${
-              JSON.stringify(prop.data.defaultValue)
-            })\n`
-            : ""
-        }` +
-        `${entryBlock}` +
-        `${indent(indent_level)}.build()`;
+      return generatePropBuilderStringInner(prop.kind, entryBlock);
```

Comment:
Gathered all of the variants of generatePropBuilder() into a single generatePropBuilderStringInner(). I verified that it generates identical output (as long as you don't include the new setDocumentation / setDocLink calls).